src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -538,3 +540,179 @@ export const sampleData: RawSpeciesStats = {
     }
   }
 };
+
+type SpeciesSidebarData = {
+  [genomeId: string]: SpeciesSidebarPayload;
+};
+
+export const sidebarData: SpeciesSidebarData = {
+  homo_sapiens_GCA_000001405_28: {
+    id: 'GCA_000001405.28',
+    taxonomy_id: '9606',
+    strain: null,
+    database_version: '100.38',
+    common_name: 'Human',
+    scientific_name: 'Homo sapiens',
+    gencode_version: 'GENCODE 34',
+    assembly_name: 'GRCh38.p13',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Ensembl',
+      url: 'www.ensembl.org'
```

Comment:
Probably want to add `https://`; without the `//` part the link behaves as a relative link.

Don't know what should happen when the urls start arriving via an api. Ideally, the backend should also know the protocol for reaching this resource; we shouldn't be guessing it on the frontend. Alternatively, we can use `http` everywhere, and let the target resources redirect to `https` if they so wish.

Reply:
I've added `http://` to the missing ones